ruby26/README.md

```diff
@@ -0,0 +1,15 @@
+# ruby26
+
+A dynamic, open source programming language with a focus on   simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.
+
+## Maintainers
+
+* The Habitat Maintainers: <humans@habitat.sh>
+
+## Type of Package
+
+Binary package
+
+## Usage
+
+*TODO: Add instructions for usage*
```

Comment:
TODO's were used to backfill requirements for readme contents. If we're adding a new plan, we really should complete the readme with usage instructions 😄  Other than this, the PR looks great!

Reply:
Do you have an example of a good usage section for an app plan. I just copied this readme straight from the previous ruby version.